### ISSUE TITLE
Plans to support influxdb 0.9.x?

### ISSUE THREAD

**Issue by rberger** [bdangit/chef-influxdb#44]
Is anyone working on modifying chef-influxdb to support influxdb 0.9.x?

Thanks

**Comment by jakedavis** [comment]
Hey @rberger, could you elaborate a bit more on what you mean here? Are there backwards-incompatible changes you're anticipating? I'm not familiar with the 0.9.x release at present, I admit.

**Comment by rberger** [comment]
The Configuration file has changed some. I think that may be it. 

I'm taking a stab at making it work. Will send a pull request if I get it to work. Hopefully make it work with pre and post 0.9.x, or will send questions if I can't make it work :-)

Trying to figure out how to set `default[:influxdb][:config]` differently for the different versions but still allow attribute overrides of components `default[:influxdb][:config]`  in wrapper cookbooks / roles, etc and make it backwards compatible.

Was mainly wondering if anyone else has tried to make the cookbook work before I get too deep in my own attempt.

**Comment by jakedavis** [comment]
Ahhh okay. Well yeah a PR would be super! There are no efforts on our front, anyway. Trying to think about the "feature flip" there, but I admit I'm out of ideas. I'll continue thinking about it today and report back if I've got an idea.

**Comment by cmjosh** [comment]
The major blocker for using this cookbook for 0.9.0 right now is the influxdb gem which most of the LWRP's are built on. It does not yet support 0.9.x. Source: https://github.com/influxdb/influxdb-ruby

Aside from a minor config issue, installing the 0.9.0 package & starting the service seems to work fine, but any of the other LWRP's that use the gem will probably be broken. 
 
I know for sure that ``influxdb_database`` doesn't work - it returns "404 page not found" when trying to call ``get_database_list``.  I wouldn't be surprised if the other resources have problems also with all of the API changes that were made.

**Comment by kri5** [comment]
influxdb/influxdb-ruby#77 should be closed soon, it might unblock support for 0.9.x

**Comment by kri5** [comment]
influxdb/influxdb-ruby#92 as been merged into master, so now influxdb-ruby should support 0.9.x .
Is there anything else blocker?

**Comment by cmjosh** [comment]
The 0.2.x versions of influxdb-ruby work with influxdb 0.9.x now, but the gem API was changed in the process.  https://github.com/SimpleFinance/chef-influxdb/pull/66 addresses additional changes in the LWRP's that would be needed.

**Comment by kri5** [comment]
Any updates?

**Comment by jperville** [comment]
+1

**Comment by jakedavis** [comment]
Hi everyone!

Happy to review that PR and will do so today. Apologies that it's been sitting there, my fault. In a strange twist, we don't actually use this cookbook or InfluxDB at all! We experimented with it (hence this cookbook), but never ended up deploying it. With that said, we could use a contributor or two if someone is interested in that role. If not, that works too, but review will be a bit slower. Sorry for the delay and thanks for understanding!

**Comment by bdangit** [comment]
bleeding edge as far as I know supports 0.9.x. reopen if there are issues.
